jenkinsapi_tests/unittests/test_requester.py

```diff
@@ -362,12 +362,13 @@ def fake_post(*args, **kwargs):  # pylint: disable=unused-argument
     monkeypatch.setattr(requests.Session, 'post', fake_post)
 
     req = Requester('foo', 'bar')
-    with pytest.raises(JenkinsAPIException):
+    with pytest.raises(JenkinsAPIException) as error:
         req.post_and_confirm_status(
             url='http://dummy',
             params={'param': 'value'},
             data='some data'
         )
+    assert 'status=500' in str(error)
```

Comment:
Not sure where `status=500` will come from...

Reply:
From here: https://github.com/pycontribs/jenkinsapi/blob/master/jenkinsapi/utils/requester.py#L182